android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -62,11 +54,6 @@
     <string name="title_activity_oob">OobActivity</string>
```

Comment:
It doesn't appear to be used for anything the user sees. I will just move it to the other file.